.github/workflows/iroha2-dev-pr.yml

```diff
@@ -76,8 +76,7 @@ jobs:
           path: ${{ env.DOCKER_COMPOSE_PATH }}
       - name: Run tests, with coverage
         run: |
-          cargo build --bin irohad
-          export IROHAD_EXEC=$(realpath ./target/debug/irohad)
+          mold --run cargo install --path ./crates/irohad 
```

Comment:
If we end up fixing the cache, this will likely break without `which irohad ||` because `cargo install` returns an error if the binary has been installed.

Reply:
Will do, but it doesn't currently work anyway =(

I have yet to investigate (logs take 200mb) why

Reply:
Added `which irohad` check

`cargo install` works, by the way. But I got a vast amount of "multiple items" errors in rust std while it compiles wasms/ui tests, haven't figured it out yet.